xDSCResourceDesigner.psm1

```diff
@@ -2242,152 +2334,162 @@ function Test-MockSchema
         $errorIdsRef
     )
 
-    # Returns full path to a 0 byte .tmp file
-    $tempFilePath = [IO.Path]::GetTempFileName() 
+    $newSchemaPath = $null
+    $newSchemaName = $null
 
-    $tempFolderPath = [IO.Path]::GetDirectoryName($tempFilePath)
-
-    # Extracts the ????.tmp name
-    $newSchemaName = [IO.Path]::GetFileNameWithoutExtension($tempFilePath)
+    try
+    {
+        # Returns full path to a 0 byte .tmp file
+        $tempFilePath = [IO.Path]::GetTempFileName() 
 
-    # We can now use the temp file name to create a new unique file
-    Remove-Item $tempFilePath
+        $tempFolderPath = [IO.Path]::GetDirectoryName($tempFilePath)
 
-    $newSchemaPath = "$tempFolderPath\$newSchemaName.schema.mof"
+        # Extracts the ????.tmp name
+        $newSchemaName = [IO.Path]::GetFileNameWithoutExtension($tempFilePath)
 
-    $null = New-Item $newSchemaPath -ItemType File
+        # We can now use the temp file name to create a new unique file
+        Remove-Item $tempFilePath
 
-    $null = [IO.Path]::GetFileNameWithoutExtension($Schema) -cmatch "(.+)\.schema"
-    $schemaName = $Matches[1]
+        $newSchemaPath = "$tempFolderPath\$newSchemaName.schema.mof"
 
-    # Initialize this to correct; it will be overwritten if incorrect
-    $className = $schemaName
+        $null = New-Item $newSchemaPath -ItemType File
 
-    $extendsOMI = $false
+        $null = [IO.Path]::GetFileNameWithoutExtension($Schema) -cmatch "(.+)\.schema"
```

Comment:
I agree, but I didn't change that line other than to indent it in the try {} block.  :)  Unfortunately the git diff is not great at handling this section, but if you compare the files in something like WinMerge, it looks much more tidy.